### ISSUE TITLE
ocaml-lsp-server 1.9.0 doesn't support ocamlformat-rpc-lib 0.21.0

### ISSUE THREAD

**Issue by Alizter** [ocaml/ocaml-lsp#637]
The following came up when installing ocamlformat-rpc:
```
  ↗ upgrade   ocamlformat-rpc-lib 0.19.0 to 0.21.0 [required by ocamlformat-rpc]
  ↘ downgrade ocaml-lsp-server    1.9.0 to 1.7.0
  ∗ install   ocamlformat-rpc     0.21.0
```
I can't tell if this is an issue with ocamlformat or lsp however. I would have expected lsp-server to work fine with newer versions of ocamlformat-rpc-lib.

**Comment by ulugbekna** [comment]
@panglesd hi, do you have time to have a look at this?

**Comment by panglesd** [comment]
Yes I will have a look.

**Comment by panglesd** [comment]
I think the maximal version of `ocamlformat-rpc-lib` could be updated to `0.20.0` right now, but as soon as #599 is merged, `ocaml-lsp` will require `ocamlformat-rpc-lib >= 0.21.0`.

**Comment by rgrinberg** [comment]
Fixed in 1.10.5
